xray/commands/audit/sca/java/mvn.go

```diff
@@ -148,7 +147,7 @@ func (mdt *MavenDepTreeManager) RunMvnCmd(goals []string) ([]byte, error) {
 // Creates a new settings.xml file configured with the provided server and repository from the current MavenDepTreeManager instance.
 // The settings.xml will be written to the given path.
 func (mdt *MavenDepTreeManager) createSettingsXmlWithConfiguredArtifactory(path string) error {
-	serverAuthentication, err := mdt.getSettingsXmlServerAuthentication()
+	username, password, err := mdt.server.GetAuthenticationCredentials()
```

Comment:
Can we return an error if username is empty,? Mentioning that only username+password/token is supported